WearSpeakerSample/README.md

```diff
@@ -1,11 +1,14 @@
 
-Android WearSpeakerSample Sample
+WearSpeakerSample
 ================================
 
 A sample that shows how you can record voice using the microphone on a wearable and
 play the recorded voice or an mp3 file, if the wearable device is connected to a speaker
 (bluetooth or built-in).
 
+This sample is also written entirely with Jetpack Compose, using [Wear-specific components provided
+by AndroidX](https://developer.android.com/jetpack/androidx/releases/wear-compose).
```

Comment:
We should call out this is an advanced Compose sample and why that is so devs who find themselves reading this README really understand that.

Reply:
I think what you've described is a call to ask the compose core team how to implement this without that extra non obvious code. If its something missing in compose maybe we can get a few methods added or a different way to achieve it.